homeassistant/components/sma_manager/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""THE SMA Manager integration."""
```

Comment:
Sorry about this, I didn't mean for all caps 🙈 
```suggestion
"""The SMA Manager integration."""
```